UI/window-basic-settings.cpp

```diff
@@ -6325,6 +6325,10 @@ void OBSBasicSettings::UpdateMultitrackVideo()
 			ui->enableMultitrackVideo->setChecked(false);
 	}
 
+#ifndef _WIN32
+	available = available && MultitrackVideoDeveloperModeEnabled();
```

Comment:
there's an argument to be made about changing this to
```suggestion
	available = IsCustomService() && MultitrackVideoDeveloperModeEnabled();
```
instead, but I think it's still potentially useful as is in some cases (e.g. someone developing their own configuration service and testing hitting that endpoint from linux?)

Reply:
As @derrod said, this seems fine as-is.